dso-l2/src/main/java/com/tc/services/TerracottaServiceProviderRegistryImpl.java

```diff
@@ -53,27 +53,12 @@ public void initialize(String serverName, TcConfiguration configuration) {
             registerNewServiceProvider(provider);
           }
         } catch (InstantiationException | IllegalAccessException ie) {
-//  really shouldn't be doing this.  ServiceProvider configurations should provide appropriate classes
-//  to handle the config.
-          tryServiceLoader(config);
+          logger.error("caught exception while initializing service " + serviceClazz, ie);
```

Comment:
Should this re-throw and crash the server or just fail to load the service?  I'm inclined to say this should re-throw and crash.  Fail fast.